app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -27,7 +27,7 @@ function show_interface_modal(modal_content) {
   modal_window
     .find('.modal-title')
     .text(__('Interface') + ' ' + String(identifier));
-  modal_window.modal({ show: true });
+  modal_window.modal({ show: true , backdrop: 'static'});
```

Comment:
Somehow because of the select parent change, if its inside a modal, there is, a pixel in the select in the modal, that if a user clicks on at the right timing, closes the modal as its somehow outside the modal.
This makes is so the modal isnt closed on a click out side (only by clicking the X/save/cancel buttons)
other modal in the hosts page already has it `$('#host-conflicts-modal').modal({ show: 'true', backdrop: 'static' });`

Reply:
The tests, while running in normal, not headless mode, some how click this pixel in the second select in the interface modal, and close it